tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: include os-specific variables
-  include_vars: "{{ ansible_os_family }}.yml"
+  include_vars: "{{ ansible_os_family | lower }}.yml"
```

Comment:
I think we can just remove this task considering that now we don't depend anymore on any OS specific variable.